core/src/main/java/pl/agh/edu/model/Room.java

```diff
@@ -44,42 +95,34 @@ public void setRank(RoomRank rank) {
 		this.rank = rank;
 	}
 
-	public RoomState getState() {
-		return state;
-	}
-
-	public void setState(RoomState state) {
-		this.state = state;
-	}
-
 	public int getCapacity() {
 		return capacity;
 	}
 
 	public boolean upgradeRank() {
```

Comment:
to raczej powino być void , a to sprawdzenie powino się odbywać w checku w osobnej funkcji , albo w ostateczności zmienić nazwę fukcji na coś pokroju tryUpgradeRank() bo nie sugeruje w ogóle że coś zwróci